tests/conftest.py

```diff
@@ -41,7 +41,21 @@ def form_data_model(fixture, user):
         'elements_css_classes': 'form-control',
         'elements': json.loads(fixture('simple_form.json'))['form']['elements'],
     }
-    return FormData.objects.create(**form_data)
+    return model_data
+
+
+@pytest.fixture
+def model(model_data, user):
+    data = model_data.copy()
```

Comment:
Because the name points to `model_data` and I don't want to change the originally `author` with the User-id.